test/verify/check-system-journal

```diff
@@ -329,6 +329,31 @@ ExecStart=/bin/sh -c 'sleep 5; for s in $(seq 10); do echo SLOW; sleep 0.1; done
 
         self.allow_journal_messages(".*Data from the specified boot .* is not available: No such boot ID in journal.*")
 
+    def testIdentifierReloading(self):
+        b = self.browser
+        m = self.machine
+        tag_error = "check-error-inside"
+        tag_info = "check-info-inside"
+        m.execute(f"logger -p user.err --tag {tag_error} {tag_error.upper()}")
+        m.execute(f"logger -p 5 --tag {tag_info} {tag_info.upper()}")
+
+        self.login_and_go("/system/logs")
+        self.browser.select_PF4("#journal-prio-menu", "Info and above")
+        b.wait_visible(f"#journal-box .cockpit-logline .cockpit-log-message:contains('{tag_error.upper()}')")
+        b.wait_visible(f"#journal-box .cockpit-logline .cockpit-log-message:contains('{tag_info.upper()}')")
+
+        self.browser.select_PF4("#journal-prio-menu", "Error and above")
+        b.reload()
+        b.enter_page("/system/logs")
+        b.select_PF4("#journal-identifier-menu .pf-v5-c-select__toggle", tag_error)
+        b.wait_not_present(f"#journal-identifier-menu + ul button:contains('{tag_info}')")
```

Comment:
Maybe better to make sure the other is present

Reply:
`b.select_PF4` do just select in case of present. so this test is there.